ios/FluentUI.Demo/FluentUIDemoTests/ActivityIndicatorTest.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import XCTest
+
+class ActivityIndicatorTest: XCTestCase {
+    let app = XCUIApplication()
+
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        continueAfterFailure = false
+        app.launch()
+
+        let onHomePage = app.staticTexts["FluentUI DEV"].exists
+        let onDifferentControlPage = !onHomePage && !app.staticTexts["ActivityIndicator"].exists
+        let activityIndicatorCell = app.staticTexts["ActivityIndicator"]
```

Comment:
It's a little confusing since the two staticTexts are different (one is the staticText that is each cell on the home page and the other is the actual title of each control page) but I will try!

Reply:
I think what you updated to is what I had in mind :)